plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -134,7 +125,8 @@ export class PgInstrumentation extends InstrumentationBase {
             span = utils.handleParameterizedQuery.call(
               this,
               plugin.tracer,
-              plugin._config as InstrumentationConfig & PgInstrumentationConfig,
+              plugin.getConfig() as PgInstrumentationConfig &
```

Comment:
if `PgInstrumentationConfig` extends `InstrumentationConfig` why do you need to cast it as `PgInstrumentationConfig && InstrumentationConfig` ?
`PgInstrumentationConfig` should be enough.

Reply:
I agree and also wondered about that. I left it as is to minimize my code changes but could also change it.

Reply:
@obecny, I modified the casting 👍🏻